### PR TITLE
small small widget changes

### PR DESCRIPTION
**Pull request by paigemiller**
addressing https://github.com/CDCgov/cfa-ngm-widget/issues/57

### CODE REVIEW

File: scripts/widget.py [hunk 1]
```diff
@@ -36,10 +36,10 @@ def summarize_scenario(
     # Run the simulation with vaccination
     result = simulate_scenario(params, distributions_as_percents=True)
 
-    st.header(f"**{params['scenario_title']}**")
+    st.header(f"*{params['scenario_title']}*")
 
-    prop_vax_help = f"Vaccination of 100% does not guarantee complete immunity if VE is less than 1. VE is {params['ve']}"
-    st.subheader("Percent of each group vaccinated:", help=prop_vax_help)
+    prop_vax_help = f"Based on allocated doses, what percent of each group is successfully immune? In the counter factual scenario, we assume no vaccines are administered. Vaccination of 100% does not guarantee complete immunity if VE is less than 1. VE is {params['ve']}"
```
Comment (afmagee42) [inline scripts/widget.py:41]: I'm happy for us to report successful vaccinations but that is not what is currently being computed.

E.g. with default of 500k core getting 250k vaccines at VE 0.74, this reports 50% vaccination.

File: scripts/widget.py [hunk 2]
```diff
@@ -184,6 +177,15 @@ def app():
         counterfactual,
     ]
 
+    # text outside scenarios
+    summary_help = (
```
Comment (afmagee42) [inline scripts/widget.py:181]: I like moving this out and only printing it once. Reduces bloat.

**Review by afmagee42 — CHANGES_REQUESTED** [review]
I like the changes but we need to align text and computation re: % vaccinated

**Review by swo — APPROVED** [review]
LGTM, but I defer to @paigemiller and @afmagee42

**Comment by swo** [comment]
@afmagee42 I'm merging on @paigemiller 's request; if you have further edits, we can do them in a future PR

**Pull request merged**